roadmap/implementers-guide/src/runtime/disputes2.md

```diff
@@ -0,0 +1,57 @@
+# Disputes
+
+## Motivation
+
+If the transaction can be verified the defined verification code, it is good. If not, there is a dispute
+about a block, which was either detected during the backing stage by a backing checker
+or, made it through the backing stage, but was then found invalid by an approval checker.
+
+(If there was no conflict between votes, but nobody backed the block in the first place, it is TBD if that falls in the category of a dispute or not)
```

Comment:
dispute implies conflict IMO. If there's nobody to slash because a backer just rejected some bad collator's block, then we have nobody to slash anyway.